src/net.h

```diff
@@ -640,8 +640,14 @@ enum
     LOCAL_MAX
 };
 
-bool IsPeerAddrLocalGood(CNode *pnode);
-void AdvertiseLocal(CNode *pnode);
+bool IsPeerAddrLocalGood(CNode& pnode);
+
+/**
+ * Add our "best" local address (see GetLocal) to the batch of addresses
+ * we are planning to relay to a given node as scheduled.
+ * The relay is not guaranteed.
```

Comment:
I suggest removing this. The fact that addresses added to `vAddrToSend` are not guaranteed to be sent is not a property of `AdvertiseLocal`, so the comment shouldn't be here.